src/screen/trip.js

```diff
@@ -1,18 +1,91 @@
-import React from "react";
+import React, {useState} from "react";
```

Comment:
you can import `useEffect` here so you don't have to write `React.useEffect()` everytime. 

Also, React is already imported at the global scope for all `/src` files for React v16+ (I could be technically wrong here, what I mean is you don' need to import `React` explicitly in `/src/` files.